applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -116,12 +121,13 @@ Direct-xip mode
 
 The direct-xip mode is used for the :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`.
 In this mode, the MCUboot bootloader boots an image directly from a given slot, so the swap operation is not needed.
-To set the MCUboot mode of operations to the direct-xip mode, make sure to enable the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` Kconfig option in the application configuration.
-This option automatically enables :kconfig:option:`CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT` to build application update images for both slots.
+To set the MCUboot mode of operations to the direct-xip mode, make sure to enable the :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP` Kconfig option in the sysbuild configuration.
+This option automatically enables :kconfig:option:`SB_CONFIG_MCUBOOT_BUILD_DIRECT_XIP_VARIANT` to build application update images for both slots.
+The nRF Desktop application do not enable the direct-xip mode with revert mechanism (:kconfig:option:`SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT`).
 
-Enable the ``CONFIG_BOOT_DIRECT_XIP`` Kconfig option in the bootloader configuration to make the MCUboot run the image directly from both image slots.
-The nRF Desktop's bootloader configurations do not enable the revert mechanism (``CONFIG_BOOT_DIRECT_XIP_REVERT``).
-When the direct-xip mode is enabled (the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` Kconfig option is set in the application configuration), the application modules that control the DFU transport do not request firmware upgrades and do not confirm the running image.
+The ``CONFIG_BOOT_DIRECT_XIP`` Kconfig option is automatically applied to the bootloader configuration based on the sysbuild configuration to make the MCUboot run the image directly from both image slots.
+Similarly, the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` Kconfig option is automatically applied to the application configuration based on the sysbuild configuration to inform the application in which mode the MCUboot bootloader works.
+When the direct-xip mode is enabled, the application modules that control the DFU transport do not request firmware upgrades and do not confirm the running image.
 In that scenario, the MCUboot bootloader simply boots the image with the higher image version.
```

Comment:
Can we also update MCUboot with serial recovery doc? (we also need to set some Sysbuild configurations in that case)